raiden/raiden_service.py

```diff
@@ -415,17 +415,7 @@ def start(self) -> None:
         self.install_all_blockchain_filters(
             self.default_registry, self.default_secret_registry, last_log_block_number
         )
-
-        # Complete the first_run of the alarm task and synchronize with the
-        # blockchain since the last run.
-        #
-        # Notes about setup order:
-        # - The filters must be polled after the node state has been primed,
-        # otherwise the state changes won't have effect.
-        # - The alarm must complete its first run before the transport is started,
-        #   to reject messages for closed/settled channels.
-        self.alarm.register_callback(self._callback_new_block)
-        self.alarm.first_run(last_log_block_number)
+        self._prepare_and_execute_alarm_first_run(from_block=last_log_block_number)
 
         chain_state = views.state_from_raiden(self)
```

Comment:
Please, add a comment here saying that this must be done after the alarm task ran and the wal was restored..